### ISSUE TITLE
Please ship new gnome-keyring!

### ISSUE THREAD

**Issue by delmartechdude** [pop-os/pop#206]
ed25519 keys are completely unusuable on Pop!_OS!

 **Distribution - (run ```cat /etc/os-release```)**
```
NAME="Pop!_OS"
VERSION="17.10 (Artful Aardvark)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 17.10 (Artful Aardvark)"
VERSION_ID="17.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com/"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=artful
UBUNTU_CODENAME=artful
```
 
 **Related Application and/or Package Version - apt policy $PACKAGE NAME**
```
gnome-keyring:
  Installed: 3.20.1-1ubuntu1
  Candidate: 3.20.1-1ubuntu1
  Version table:
 *** 3.20.1-1ubuntu1 500
        500 http://us.archive.ubuntu.com/ubuntu artful/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Issue/Bug Description**
Can't use ed25519 keys.

**Steps to reproduce (if you know)**
`ssh-add <ed25519 key>`

**Expected behavior**
imports fine.

**Other Notes**
So our gnome-keyring is 9 months old. Version `3.27.92` has the fix

This was already fixed here:
https://git.gnome.org/browse/gnome-keyring/commit/?id=8db2b97

As part of this bug:
https://bugzilla.gnome.org/show_bug.cgi?id=775981

**Comment by jackpot51** [comment]
This is an issue you should take up with Ubuntu here: https://bugs.launchpad.net/ubuntu/+source/gnome-keyring

**Comment by delmartechdude** [comment]
Upgrading to Pop!_OS 18.04 beta fixed everything! Since the patch was included in Ubuntu 18.04. It blows my mind that people dealt with not being able to use ed25519 or ECDSA keys on Ubuntu + GNOME before Ubuntu 18.04. Unreal. Every other OS works fine with it. At least it is fixed now!
